lib/github_changelog_generator/generator/generator_processor.rb

```diff
@@ -63,6 +63,36 @@ def remove_issues_in_milestones(filtered_issues)
       end
     end
 
+    # Method filter issues, that belong only specified tag range
+    # @param [Array] issues issues to filter
+    # @param [Symbol] hash_key key of date value default is :actual_date
+    # @param [String] older_tag all issues including and before this tag's sha will be excluded. May be nil, if it's first tag
+    # @param [String] newer_tag all issue after this tag sha will be excluded. May be nil for unreleased section
+    # @return [Array] filtered issues
+    def delete_by_sha(issues, older_tag = nil, newer_tag = nil)
+      # in case no tags specified - return unchanged array
+      return issues if older_tag.nil? && newer_tag.nil?
+
+      older_tag = ensure_older_tag(older_tag, newer_tag)
+
+      issues.select do |issue|
+        # XXX It would be better to just grab merge_commit_sha but the way GCG
+        # fetches PRs currently doesn't have that info. See:
+        # https://developer.github.com/v3/pulls/#get-a-single-pull-request
+        if issue["events"] && (event = issue["events"].find { |e| e["event"] == "merged" })
+          sha = event["commit_id"]
+
+          sha_between_tags?(sha, newer_tag, older_tag)
```

Comment:
This should now use the `issue["first_occurring_tag"]` as it has already been fetched.